crates/goose/Cargo.toml

```diff
@@ -101,6 +101,7 @@ unicode-normalization = "0.1"
 
 arrow = "52.2"
 oauth2 = "5.0.0"
+goose-mcp = { version = "1.8.0", path = "../goose-mcp" }
```

Comment:
hrm - there was https://github.com/block/goose/pull/4681 change by @jamadeo  which IIUC - this would end up circular hrm. Not sure what direction dependency would be but this seems the wrong way?

Reply:
ie the core shouldn't depend on mcp but other things should depend on core?